### ISSUE TITLE
long term: install in a known spot

### ISSUE THREAD

**Issue by srl295** [unicode-org/full-icu-npm#6]
If you came here from the warning after installing `full-icu` please read:
- Long term solution is: https://github.com/nodejs/node/issues/3460 (Feedback wanted)
- `full-icu` should detect that 3460 (above) has been merged (by node version) and do whatever needs to happen to write the dat file in the right spot.
- I'm not intending to deprecate `full-icu`, but to make it "do the right thing" going forward.

**Comment by icambron** [comment]
I'm a little confused. If I understood those instructions right, I just have to run that copy command and then when I run `node` I should have full ICU support? That's not what's happening for me at all. I'm on Node 8.1.2 and I'm getting:

```js
new Intl.DateTimeFormat('es',{month:'long'}).format(new Date(9E8)) //=> "M01"
```

There are some other peculiarities: the instructions tell me to copy the file to `./node-modules/.node-icu` but `require('full-icu').datPath()` give me `/home/travis/build/icambron/luxon/icudt59l.dat`. I'm just copying the file to both places, but nothing is working.

You can see my flailing here: https://travis-ci.org/icambron/luxon/builds/246657525. Am I just missing some key step?

**Comment by srl295** [comment]
> Edit: nevermind, from searching around I discovered that the ICU auto-detection thing is sorta bunk and that I need to explicitly point Node at the ICU package.

It's not bunk, it's just vaporware so far. There's a difference and help wanted @ https://github.com/nodejs/node/issues/3460

**Comment by icambron** [comment]
Would you accept a smaller PR that just updated the instructions to tell the user to set the env var?

**Comment by icambron** [comment]
Ah, my bad: I hadn't rechecked the console instructions since writing my original comment above. I see now that the instructions have since been fixed by #15, which I guess is all I was suggesting doing in my vaguely-proposed PR. So all my confusion here is actually fixed by the updated console instructions. Sorry about the noise.
